recipe/meta.yaml

```diff
@@ -13,9 +13,10 @@ source:
     - patches/0003-properly-point-to-triton-includes.patch
     - patches/0004-unvendor-packages.patch
     - patches/0005-fix-tablegen.patch
+    - patches/0006-remove-third-party-references.patch
```

Comment:
Not all the patches are included here.

Also, which one is the one from upstream? You're listed as the author of all of them... Would be good to leave a link to the upstream commit/pr in a comment above the patch here.

Reply:
PS. Here's a [guide](https://github.com/conda-forge/ray-packages-feedstock/blob/main/ray-patching.md) for rebasing patches and keeping attribution etc.

Reply:
None of them are upstream anymore after following @hmaarrfk's suggestion to use CONDA_PREFIX instead of their environment variable. A few patches were previously already in the feedstock as "raw diffs" instead of patches with attribution. I am happy to remove my attribution from them, but I don't have time to dig into who came up with them originally.

Reply:
I removed my authorship when author is unknown - this represents the current state in the feedstock. Feel free to update the patches if you know who authored the patches.

Reply:
I tend to go to the file in the GH UI, then history, then figure out who committed it first and use that author & date for the commit.

Reply:
All done